xmtp/src/conversation.rs

```diff
@@ -140,11 +157,30 @@ where
             self.client.account.addr(),
             content_bytes,
             MessageState::Unprocessed as i32,
+            now(),
         )
         .store(&mut self.client.store.conn().unwrap())?;
         Ok(())
     }
 
+    pub async fn list_messages(
```

Comment:
I made this async so that we can refresh the message store, once there is a way to do that from inside a conversation.